system-variables.md

```diff
@@ -3440,7 +3440,7 @@ For a system upgraded to v5.0 from an earlier version, if you have not modified
 - Default value: `-1`
 - Range: `[-1, 256]`
 - Unit: Threads
-- This variable is used to set the maximum concurrency for TiFlash to execute a request. The default value is `-1`, indicating that this system variable is invalid. When the value is `0`, the maximum number of threads is automatically configured by TiFlash.
+- This variable is used to set the maximum concurrency for TiFlash to execute a request. The default value is `-1`, indicating that this system variable is invalid and depends on the setting of the TiFlash configuration `profiles.default.max_threads`. When the value is `0`, the maximum number of threads is automatically configured by TiFlash.
```

Comment:
```suggestion
- This variable is used to set the maximum concurrency for TiFlash to execute a request. The default value is `-1`, indicating that this system variable is invalid and depends on the setting of [`tidb_max_tiflash_threads`](/system-variables.md#tidb_max_tiflash_threads-new-in-v610). When the value is `0`, the maximum number of threads is automatically configured by TiFlash.
```

Starting from v6.1.0, the `profiles.default.max_threads` configuration item is transformed to `tidb_max_tiflash_threads`


```
    * Transform the TiFlash configuration item `max_threads` to a system variable `tidb_max_tiflash_threads`, so that the configuration can be modified dynamically and persisted. Note that the original configuration item remains after transformation.
```

Reply:
@windtalker Could you please help review this PR? Thanks!

Reply:
Hi, this description for `-1` means if `tidb_max_tiflash_threads` is `-1`, then this variable is ignored, and the max concurrency of TiFlash is decided by TiFlash's own configuration `profiles.default.max_threads`. So the original doc is the correctly one.